app/frontend/src/features/profile/view/Overview.tsx

```diff
@@ -37,38 +37,42 @@ import makeStyles from "utils/makeStyles";
 const useStyles = makeStyles((theme) => ({
   card: {
     flexShrink: 0,
-    margin: theme.spacing(2),
-    padding: theme.spacing(2),
-    width: "25%",
-    [theme.breakpoints.down("md")]: {
-      marginBottom: theme.spacing(2),
-    },
+    borderRadius: theme.shape.borderRadius * 2,
+    padding: theme.spacing(3),
     [theme.breakpoints.down("sm")]: {
-      margin: 0,
       marginBottom: theme.spacing(1),
       width: "100%",
     },
   },
+
   cardActions: {
     flexWrap: "wrap",
     justifyContent: "center",
-    paddingLeft: 0,
-    paddingRight: 0,
-    paddingTop: 0,
+    padding: theme.spacing(0.5),
     "& > *": {
-      marginBlockStart: theme.spacing(1),
+      margin: theme.spacing(0.5),
     },
   },
-  grow: {
-    paddingTop: "100%",
-  },
+
   info: {
-    marginTop: theme.spacing(0.5),
+    marginTop: theme.spacing(1),
   },
+
   intro: {
     display: "flex",
     justifyContent: "center",
   },
+
+  wrapper: {
+    marginTop: theme.spacing(1),
+    "& h1": {
+      lineHeight: "3rem",
```

Comment:
Hmm, I'm a bit worried that this will subtly break if we want to change h1's size in the theme. Is there a way around that?

Reply:
If we change the h1 size we will have also to review this spacing.

Reply:
Can you use a unitless line height instead (which is a multiple of font size)?